pybamm/models/base_model.py

```diff
@@ -67,8 +68,15 @@ def __init__(self):
             "positive electrode": pybamm.Uniform1DSubMesh,
         }
 
+        self.default_spatial_methods = {
+            "negative electrode": pybamm.FiniteVolume,
+            "separator": pybamm.FiniteVolume,
+            "positive electrode": pybamm.FiniteVolume,
+        }
         self.mesh = pybamm.Mesh(self.default_geometry, submesh_types, submesh_pts)
-        self.default_discretisation = pybamm.FiniteVolumeDiscretisation(self.mesh)
+        self.default_discretisation = pybamm.Discretisation(
+            self.mesh, self.default_spatial_methods
+        )
```

Comment:
It could be left to external classes, such as `StandardModelTest`, to call `Mesh(self.default_geometry, self.default_submesh_types, self.default_submesh_pts)` and `Discretisation(mesh, self.default_spatial_methods)` (given that there is now only one `Mesh` class and only one `Discretisation` class).
Otherwise, in every model, if you want to change the default submesh types then you also have to go through and redefine `self.mesh` and then redefine `self.default_discretisation`. 
Similar comment also applies to the line `self.default_parameter_values.process_geometry(self.default_geometry)`

Reply:
Yeah probably should reformulate all this so that just call `tests.shared.testing_disc` and tests.shared.finite_volume_1D_macro_disc` etc. I'll add a new issue to do this as it's a bit of a pain.